x-aeson/test/Test/X/Data/Aeson.hs

```diff
@@ -59,8 +60,18 @@ prop_asText t =
 
 prop_as :: Text -> Property
 prop_as t =
-  asWith (parseJSON :: Value -> Parser Text) t === as t
+  discardError (asWith (parseJSON :: Value -> Parser Text) t)
+  ===
+  discardError (as t)
+
+discardError :: Either a b -> Either () b
+discardError = \case
+  Left _ ->
+    Left ()
+  Right x ->
+    Right x
 
 return []
 tests :: IO Bool
-tests = $quickCheckAll
+tests =
+  $forAllProperties . quickCheckWithResult $ stdArgs { maxSuccess = 1000 }
```

Comment:
if this is using disorder, you could use the environment thing like
```
tests = $disorderCheckEnvAll TestRunMore
```
and then it defaults to 1000, but on the command line you can do:
```
DISORDER_RUN_MORE=10000000 ./mafia test
```
to run many more.

the other options are `TestRunNormal` for 100 and `TestRunFewer` for 10